media-libs/libsdl/libsdl-1.2.15_p20210224.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit autotools flag-o-matic multilib toolchain-funcs multilib-minimal
```

Comment:
I don't see `multilib` nor `toolchain-funcs` in use here, can you confirm?

Reply:
Removed multilib and toolchain-funcs and it works without them.